src/client/client.cpp

```diff
@@ -223,16 +223,25 @@ void Client::ProcessEvents( const SystemEvents& events )
 		}
 
 		// Select weapon.
+		if( event.type == SystemEvent::Type::Wheel && event.event.wheel.dy )
+		{
+			if(event.event.wheel.dy > 0.0f) TrySwitchWeaponNext();
+			if(event.event.wheel.dy < 0.0f) TrySwitchWeaponPrevious();
+		}
+
+		if( event.type == SystemEvent::Type::MouseKey &&
+                    event.event.mouse_key.mouse_button == SystemEvent::MouseKeyEvent::Button::Middle )
+		{
+		    TrySwitchWeaponNext();
+		}
```

Comment:
It's unusual. I do not know any game, where you can switch weapons by pressing mouse wheel. Let's avoid adding such controls.

Reply:
Ok, let's keep it.
Also we need to know, how this works in Steam release of _Chasm: the Rift_. It may be a good idea to copy controls from it.